lib/galaxy/jobs/__init__.py

```diff
@@ -902,10 +902,16 @@ def get_special( ):
     def _create_working_directory( self ):
         job = self.get_job()
         try:
+            # TEMP BLOCK --- START
+            pluggedMedia = None
```

Comment:
Everywhere ``pluggedMedia`` appears as a variable name and/or function input parameter it should be called ``plugged_media``. The Python convention is to use camel case (e.g. ``PluggedMedia`` just for class names.